public/js/poll/main.js

```diff
@@ -1,7 +1,7 @@
 (function() {
 	window.Poll = {
-		load: function(data) {
-			Poll.sockets.emit.load(data.pollid, function(err, poll) {
+		load: function(pollid) {
```

Comment:
Didn't see any reason to maintain the object parameter since only the pollid was being used.